pkg/kubelet/kubelet_getters.go

```diff
@@ -194,6 +194,11 @@ func (kl *Kubelet) getRuntime() kubecontainer.Runtime {
 	return kl.containerRuntime
 }
 
+// getRemappedIDs returns the uid and gid from the host usernamespace that are mapped to container user-namespace
+func (kl *Kubelet) getRemappedIDs() (int, int) {
```

Comment:
I'm not following what the purpose of this function is. It also seems easy to use incorrectly.

* the function name doesn't indicate whether the returned ids are in the host or container namespace
* there's no indication whether mapping is enabled or not and no ability to return an error. does this return 0,0 in that case?
* it's not clear what uid/gid are going to be returned here, since there's no container-specific input...
* there's only a single gid provided here... what is the interaction with multi-group contexts (pods with fsGroup, supplementalGroups, etc)?

Reply:
> the function name doesn't indicate whether the returned ids are in the host or container namespace

function comment is mentioning this. Will try to make it more clear.
L#197 // getRemappedIDs returns the uid and gid from the **host usernamespace**

Reply:
would something like this be clearer?
```
func (kl *Kubelet) getHostUID(containerUID int) (int, error)
func (kl *Kubelet) getHostGID(containerGID int) (int, error)
```

currently, the inputs are implicit (your comment in https://github.com/kubernetes/kubernetes/pull/64005#discussion_r203762631 explains this is returning the host uids that map to uid/gid 0, but that isn't apparent to callers of this function)

that also makes it clear when there's no host id mapped to the specified container id (rather than relying on zero values or -1 to indicate errors)

Reply:
sure